docs/data/pages.ts

```diff
@@ -328,9 +347,64 @@ const pages: MuiPage[] = [
             pathname: '/x/api/date-pickers/mobile-date-time-picker',
             title: 'MobileDateTimePicker',
           },
+          {
+            pathname: '/x/api/date-pickers/mobile-next-date-picker',
+            title: 'MobileNextDatePicker',
+          },
+          {
+            pathname: '/x/api/date-pickers/mobile-next-date-range-picker',
+            title: 'MobileNextDateRangePicker',
+            plan: 'pro',
+          },
+          {
+            pathname: '/x/api/date-pickers/mobile-next-date-time-picker',
+            title: 'MobileNextDateTimePicker',
+          },
+          {
+            pathname: '/x/api/date-pickers/mobile-next-time-picker',
+            title: 'MobileNextTimePicker',
+          },
           { pathname: '/x/api/date-pickers/mobile-time-picker', title: 'MobileTimePicker' },
           { pathname: '/x/api/date-pickers/month-calendar', title: 'MonthCalendar' },
+          {
+            pathname: '/x/api/date-pickers/multi-input-date-range-field',
+            title: 'MultiInputDateRangeField',
+            plan: 'pro',
+          },
+          {
+            pathname: '/x/api/date-pickers/multi-input-date-time-range-field',
+            title: 'MultiInputDateTimeRangeField',
+            plan: 'pro',
+          },
+          {
+            pathname: '/x/api/date-pickers/multi-input-time-range-field',
+            title: 'MultiInputTimeRangeField',
+            plan: 'pro',
+          },
+          { pathname: '/x/api/date-pickers/next-date-picker', title: 'NextDatePicker' },
+          {
+            pathname: '/x/api/date-pickers/next-date-range-picker',
+            title: 'NextDateRangePicker',
+            plan: 'pro',
+          },
+          { pathname: '/x/api/date-pickers/next-date-time-picker', title: 'NextDateTimePicker' },
+          { pathname: '/x/api/date-pickers/next-time-picker', title: 'NextTimePicker' },
           { pathname: '/x/api/date-pickers/pickers-day', title: 'PickersDay' },
+          {
+            pathname: '/x/api/date-pickers/single-input-date-range-field',
+            title: 'SingleInputDateRangeField',
+            plan: 'pro',
+          },
+          {
+            pathname: '/x/api/date-pickers/single-input-date-time-range-field',
+            title: 'SingleInputDateTimeRangeField',
+            plan: 'pro',
+          },
+          {
+            pathname: '/x/api/date-pickers/single-input-time-range-field',
+            title: 'SingleInputTimeRangeField',
+            plan: 'pro',
+          },
```

Comment:
Both of these do not exist yet.
Are they here because this PR is going to be merged only after they are implemented? 🤔

Reply:
No it's a mistake, fixed